solutions/basic/stack_definition.json

```diff
@@ -747,17 +797,13 @@
       ]
     },
     {
-      "name": "Workload - Sample RAG App Configuration",
-      "version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.b1bf8a3d-3d51-4362-a13b-eb528a38c6f3-global",
+      "name": "6 - Workload - Sample RAG App Configuration",
```

Comment:
"App Configuration" might get confused with the "App Configuration" service which is now in the stack.Perhaps just go with "Sample RAG App"?